package.json

```diff
@@ -1,27 +1,30 @@
 {
-  "name": "cout-embauche",
-  "version": "0.6.2",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:sgmap/cout-embauche.git"
-  },
-  "author": "Matti Schneider <matti.schneider@data.gouv.fr>",
-  "description": "Un simulateur de coût d'embauche",
-  "dependencies": {},
-  "devDependencies": {
-    "css-loader": "^0.18.0",
-    "expose-loader": "^0.7.0",
-    "html-loader": "^0.3.0",
-    "rimraf": "^2.4.3",
-    "style-loader": "^0.12.3",
-    "watai": "^0.6.2",
-    "webpack": "^1.12.1"
-  },
-  "scripts": {
-    "pretest": "npm run compile",
-    "test": "watai test && watai test/bootstrap-compat",
-    "precompile": "rimraf dist/*",
-    "compile": "webpack --optimize-minimize",
-    "compile-continuously": "webpack --watch"
-  }
+	"name": "cout-embauche",
```

Comment:
The `package.json` should be kept at 2-spaces indentation, as `npm` actions that modify it will always change it back to that style.